xarray/namedarray/parallelcompat.py

```diff
@@ -95,6 +95,10 @@ def guess_chunkmanager(
     """
 
     chunkmanagers = list_chunkmanagers()
+    if len(chunkmanagers) == 0:
+        raise ValueError(
```

Comment:
that would be kinda weird, though, because those usually need to point to a module that failed to import / was not found (and technically here we don't care about imports, we care about entrypoints).

Reply:
Also we could actually just special case and state the name of the module that failed to be imported.